### ISSUE TITLE
response header value changed

### ISSUE THREAD

**Issue by binbug** [rangav/thunder-client-support#1239]
**Describe the bug**

when I response a header value with `&times=1` , it change to `x=1`.

`&times` change to `x`

**To Reproduce**

server:
```go
http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("TestHeader", "&times=1")
		w.Write([]byte("hello world1"))

	})

// listen and serve
http.ListenAndServe(":8080", nil)
```

resp:
<img width="1338" alt="image" src="https://github.com/rangav/thunder-client-support/assets/53507397/d2ce608d-813b-4463-8b91-2e374b4d2abc">

postman is ok:
<img width="1122" alt="image" src="https://github.com/rangav/thunder-client-support/assets/53507397/dcd57618-27c0-48d1-8e16-406a98d37d4f">

**Expected behavior**

`&times` is `&times`

**Platform:** 
 - OS:  macos 13.0 (22A380)
 - vscode version: 1.80.1 (Universal)
 - extension version: v2.9.3

**Your Team Size Using TC:**

myself

**Comment by rangav** [comment]
Thanks @binbug for reporting the bug, will look into it.

**Comment by rangav** [comment]
This bug is fixed now and published to the marketplace, please update to `v2.10.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.10.0

Please let me know your feedback.

**Comment by binbug** [comment]
@rangav I had update to v2.10.0. And now it's ok. Thanks!
